demo.md

```diff
@@ -0,0 +1,155 @@
+# **RVIP** [Out The Box]
+**[Version]:** 1.0.0
+
+# **What Is This?**
+This document provides instruction to set-up **RVIP** as a complete Robotic Vision suite.
+
+This is based on the combination of ROS packages and external libraries used for testing purposes.
+
+This **suite** is made of:
+1. OpenGR
+2. darknet_ros
+3. rvip
+
+# **Dependencies**
+### This section lists all third-party softwares used by **RVIP**.
+
+**Click** the hyperlinks for download instructions.
+
+1. [Ubuntu 18.04 (Bionic Beaver)](https://ubuntu.com/download/desktop)
+    - For the operating system
+2. [ROS Morenia Melodic](http://wiki.ros.org/melodic/Installation/Ubuntu)
+    - For the ROS base
+3. OpenGR
+    - For pose estimation. `Refer below for download instructions.`
+4. [OpenCV 3.2.0](http://www.codebind.com/linux-tutorials/install-opencv-3-2-ubuntu-18-04-lts-linux/)
+    - For reading and displaying images
+5. [Point Cloud Library](https://askubuntu.com/questions/1134856/how-to-install-pcl-on-ubuntu-18-04)
+    - For processing 3D information
+6. darknet_ros
+    - For object recognition using only 2D RGB images. `Refer below for download instructions.`
+
+# **Connecting To Camera**
+### This section provides instructions on how you can connect **RVIP** to your ROS-supported **Depth Sensor/Camera**.  
+`Ensure that you have finished setup before doing this!`
+
+1. **Identify** **topics** and **frame_id** which your **Depth Sensor/Camera** is publishing the following information on:
+
+> `3D PointCloud`
+>
+> `Rectified 2D RGB Image`
+
+Eg. In my own use case, I use a Kinect v2 One Depth Sensor. The **topics** and **frame_id** thus look like so:
+> `/kinect2/hd/points` - For 3D PointCloud
+
+> `/kinect2/hd/image_color_rect` - For Rectified 2D RGB Image
+
+> `/kinect2_rgb_optical_frame` - For frame_id
+
+2 . **Replace** the following lines in RVIP's launch file under the specified files below:
+
+**Edit** [line 20]() of the `rvip/launch/run.launch` file to the **3D PointCloud** camera topic you identified earlier.
```

Comment:
Was `line 20` supposed to be a link?

Suggestion: show some context (ie: actual lines) instead of referring to line nrs. Line nrs change, context would help users with locating what they need to change in those cases.

Alternatively: provide additional `arg`s for your `.launch` file for the topic remaps. Then users wouldn't need to edit any files at all.

Reply:
Provided additional args for topic remaps. Documentation in **demo.md** updated to reflect change.
[commit 0ec90a8184d9177ce48f2abf0d3688c4c89c7a00](https://github.com/cardboardcode/rvip-1/commit/0ec90a8184d9177ce48f2abf0d3688c4c89c7a00)